File: app/src/main/java/com/dumuzeyn/mp3player/AlbumsMenuRenderer.java
package com.dumuzeyn.mp3player;

import java.util.ArrayList;
import java.util.Map;

final class AlbumsMenuRenderer extends TrackGroupMenuRenderer {
    AlbumsMenuRenderer(MainActivityCore host) {
        super(host);
    }

    @Override
    Map<String, ArrayList<Track>> groupedTracks() {
        return host.libraryState.homeContent.albumTracks;
    }

    @Override
    String unknownGroupName() {
        return host.tr("Unknown album", "Неизвестный альбом");
    }

    @Override
    int cardOpacity() {
        return host.appearanceState.albumCardOpacity;
    }
}

File: app/src/main/java/com/dumuzeyn/mp3player/AlbumsMenuRenderer.kt
package com.dumuzeyn.mp3player

internal class AlbumsMenuRenderer(host: MainActivityCore) : TrackGroupMenuRenderer(host) {
    override fun groupedTracks(): Map<String, ArrayList<Track>> =
        host.libraryState.homeContent.albumTracks

    override fun unknownGroupName(): String = host.tr("Unknown album", "Неизвестный альбом")

    override fun cardOpacity(): Int = host.appearanceState.albumCardOpacity
}
